mix.exs

```diff
@@ -45,11 +45,11 @@ defmodule PhoenixMicrosoftbot.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:phoenix, "~> 1.1"},
-      {:inch_ex, ">= 0.0.0", only: :docs},
-      {:ex_microsoftbot, "~> 0.1.0"},
```

Comment:
As i've published the ex_microsoftbot 1.0.0 package. Can you also make the change here to get it from hex instead of github.